service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -140,6 +142,15 @@ public int refreshExpiringPassports() {
             });
   }
 
+  private void validateRedirectUri(String redirectUri, ProviderProperties providerInfo) {
```

Comment:
What's the reason for filtering this list as opposed to looking for a match?

Reply:
I was being dense. I found `noneMatch`!